src/themes/greyVest/index.js

```diff
@@ -1,247 +1,103 @@
-        fontSize: 13,
-        color: 'initial',
-        ...(F.view(hovering) && { color: '#0076de' }),
-        ...style,
-      }}
-      {...F.domLens.hover(hovering)}
-      {...props}
-    />
-  )
-})
-ListItem.displayName = 'ListItem'
-
-export let ListGroupItem = props => (
-  <ListItem
-    style={{
-      display: 'grid',
-      gridGap: '5px',
-      gridTemplateColumns: '20px 1fr',
-      alignItems: 'center',
-    }}
-    {...props}
-  />
-)
-
-let SmallIcon = ({ icon }) => (
-  <i className="material-icons" style={{ fontSize: 20 }}>
-    {icon}
-  </i>
-)
-let iconMap = {
-  SortAscending: () => <SmallIcon icon="expand_less" />,
-  SortDescending: () => <SmallIcon icon="expand_more" />,
-  MoveLeft: () => <SmallIcon icon="chevron_left" />,
-  MoveRight: () => <SmallIcon icon="chevron_right" />,
-  RemoveColumn: () => <SmallIcon icon="remove" />,
-  AddColumn: () => <SmallIcon icon="add" />,
-  FilterExpand: () => <SmallIcon icon="filter_list" />,
-  FilterCollapse: () => <SmallIcon icon="filter_list" />,
-  FilterAdd: () => <SmallIcon icon="filter_list" />,
-  TableColumnMenu: () => (
-    <IconButton>
-      <SmallIcon icon="more_vert" />
-    </IconButton>
-  ),
-  FilterListExpand: () => <SmallIcon icon="add" />,
-  FilterListCollapse: () => <SmallIcon icon="remove" />,
-  PreviousPage: () => <SmallIcon icon="chevron_left" />,
-  NextPage: () => <SmallIcon icon="chevron_right" />,
-  Previous5Pages: () => <span>...</span>,
-  Next5Pages: () => <span>...</span>,
-  Refresh: () => (
-    <IconButton
-      className="animated pulse slow infinite"
-      style={{ animationDuration: '500ms' }}
-    >
-      <SmallIcon icon="refresh" />
-    </IconButton>
-  ),
+  Tag,
+  TagComponent: Tag,
 }
-let Icon = ({ icon, ...props }) => (
-  <Dynamic component={iconMap[icon]} {...props} />
-)
 
-let AddLabel = (
-  <Flex style={{ justifyContent: 'space-between', alignItems: 'center' }}>
-    Add Custom Filter
-    <i className="material-icons" style={{ opacity: 0.4 }}>
-      filter_list
-    </i>
-  </Flex>
-)
-
-let FilterListItem = observer(
-  ({ active, disabled, hasChildren, children, ...props }) => (
-    <div
-      style={{
-        padding: '10px 40px',
-        cursor: 'pointer',
-        fontSize: 18,
-        background: active ? '#ebebeb' : '#fff',
-        color: disabled ? '#9b9b9b' : '#000',
-      }}
-      {...props}
-    >
-      {hasChildren ? (
-        <Flex style={{ alignItems: 'center' }}>
-          {children}
-          <i className="material-icons" style={{ fontSize: 20 }}>
-            chevron_right
-          </i>
-        </Flex>
-      ) : (
-        children
-      )}
-    </div>
-  )
-)
-FilterListItem.displayName = 'FilterListItem'
+let SearchTree = () => {}
+let ButtonGroup = defaultProps({ className: 'gv-button-group' })(Flex)
 
-export let Adder = ModalFilterAdder({
+export {
+  AdderPicker as Adder,
   Button,
+  Checkbox,
+  CheckButton,
+  Fonts,
+  HighlightWrap as Wrap,
```

Comment:
We shouldn't alias these - let's just take the breaking change because the previous names weren't great